Wikipedia/Code/ArticleFetcher.swift

```diff
@@ -290,8 +290,9 @@ final public class ArticleFetcher: Fetcher, CacheFetching {
     }
     
     public func urlRequest(from url: URL, cachePolicy: WMFCachePolicy? = nil, headers: [String: String] = [:]) -> URLRequest? {
-        
-        let request = urlRequestFromPersistence(with: url, persistType: .article, cachePolicy: cachePolicy, headers: headers)
+        var requestHeaders = configuration.pageContentServiceHeaders(for: url.wmf_language)
```

Comment:
Only thing I've noticed is I think the Edit Preview endpoints need these `Accept-Language` values set as well. It doesn't look like they build their `URLRequest`s by leaning on this method.